### ISSUE TITLE
Add derived stats

### ISSUE THREAD

**Issue by Danny-Alexander** [Danny-Alexander/terratech-block-list#5]
- health/mass
 - health/price
 - storage/mass
 - storage/price
 - etc

**Comment by Danny-Alexander** [comment]
Depends on #7

**Comment by Danny-Alexander** [comment]
Superseded by #40 [Process and modify data before displaying](https://app.gitkraken.com/glo/board/XVD-R2R-2QAPHQed/card/XXDPT1VuCAAPgdI2)
